vfs/dir.go

```diff
@@ -767,15 +796,18 @@ func (d *Dir) stat(leaf string) (Node, error) {
 	}
 	item, ok := d.items[leaf]
 
-	if !ok && d.vfs.Opt.CaseInsensitive {
-		leafLower := strings.ToLower(leaf)
+	_, ci := fs.AddConfig(context.TODO())
```

Comment:
Should this be changed back to `fs.GetConfig` now? (Does it matter?)

Reply:
Yes it should be just `fs.GetConfig` if we don't need to change the config